### DISCUSSION TITLE
[Feature Request] Process Suspension Like A PsSuspend Support Or Similar

### DISCUSSION THREAD

**Discussion by JungleJango** [Flow-Launcher/Flow.Launcher#2349] [Plugin Request]
Process suspension on windows is still a mess to handle, but I still think it would be nice to have a fast and easier way to suspend and resume a process than have to use command line based apps like SysInternal's psSuspend where you need to know the process ID or the correct process name. With flow launcher that would be amazing to just Alt + Space and then select the process in a list or tap the process name initials. Yeah, I heard that some win32 processes may cause hassle when they are suspended, like block files access or such, but I still believe it would be usefull nonethless. If that could be added as a plugin for users to choose whether install it or not.

**Comment by deefrawley** [comment]
Hi, this would better be handled by a plugin as it is not a standard Windows command. So moving to the plugin request discussion board
